src/components/Pages/ManageWiki/Features/PropertiesTable.vue

```diff
@@ -0,0 +1,224 @@
+		  <template v-slot:activator="{ on, attrs }">
+			<v-btn
+			  color="primary"
+			  dark
+			  class="mb-2"
+			  v-bind="attrs"
+			  v-on="on"
+			>
+			  Add New Mapping
+			</v-btn>
+		  </template>
+		  <v-card>
+			<v-card-title>
+			  <span class="headline">{{ formTitle }}</span>
+			</v-card-title>
+
+			<v-card-text>
+			  <v-container>
+				<v-row>
+				  <v-col
+					cols="12"
+					sm="6"
+					md="4"
+				  >
+					<v-text-field
+					  v-model="editedItem.local"
+					  label="My Wikibase (Your Instance)"
+					></v-text-field>
+				  </v-col>
+				  <v-col
+					cols="12"
+					sm="6"
+					md="4"
+				  >
+					<v-text-field
+					  v-model="editedItem.wikidata"
+					  label="Wikidata"
+					></v-text-field>
+				  </v-col>
+				</v-row>
+			  </v-container>
+			</v-card-text>
+
+			<v-card-actions>
+			  <v-spacer></v-spacer>
+			  <v-btn
+				color="blue darken-1"
+				text
+				@click="close"
+			  >
+				Cancel
+			  </v-btn>
+			  <v-btn
+				color="blue darken-1"
+				text
+				@click="save"
+			  >
+				Save
+			  </v-btn>
+			</v-card-actions>
+		  </v-card>
+		</v-dialog>
+		<v-dialog v-model="dialogDelete" max-width="500px">
+		  <v-card>
+			<v-card-title class="headline">Are you sure you want to delete this mapping?</v-card-title>
+			<v-card-actions>
+			  <v-spacer></v-spacer>
+			  <v-btn color="blue darken-1" text @click="closeDelete">Cancel</v-btn>
+			  <v-btn color="blue darken-1" text @click="deleteItemConfirm">OK</v-btn>
+			  <v-spacer></v-spacer>
+			</v-card-actions>
+		  </v-card>
+		</v-dialog>
+	  </v-toolbar>
+	</template>
+	<template v-slot:[`item.actions`]="{ item }">
+	  <v-icon
+	  	small
+		class="mr-2"
+		@click="editItem(item)"
+	  >
+		edit
+	  </v-icon>
+	  
+	  <v-icon
+	  	small
+		@click="deleteItem(item)"
+	  >
+		delete
+	  </v-icon>
+	</template>
+  </v-data-table>
+</template>
+
+<script>
+export default {
+	name: 'EntityTable',
+	data: () => ({
+      dialog: false,
+      dialogDelete: false,
+      headers: [
+        {
+          text: 'My Wikibase (Your Instance)',
+          align: 'start',
+          sortable: true,
+          value: 'local',
+        },
+        { text: 'Wikidata', value: 'wikidata' },
+        { text: 'Actions', value: 'actions', sortable: false },
+      ],
+      properties: [],
+      editedIndex: -1,
+      editedItem: {
+        local: '',
+        wikidata: '',
+      },
+      defaultItem: {
+        local: '',
+        wikidata: '',
+      },
+    }),
+
+    computed: {
+      formTitle () {
+        return this.editedIndex === -1 ? 'New Mapping' : 'Edit Mapping'
+      },
+    },
+
+    watch: {
+      dialog (val) {
+        val || this.close()
+      },
+      dialogDelete (val) {
+        val || this.closeDelete()
+      },
+    },
+
+    created () {
+      this.initialize()
+    },
+
+    methods: {
+      initialize () {
+        this.properties = [
+          {
+            local: 'No Mapping',
+            wikidata: 'P31 (Instance of)',
```

Comment:
This would somehow need to be serialized as just "P31" when we send it to the API right? So we need to either make it possible to display something other than the id value (i.e. keep both label and value here) or change this to only be 'P31' :thinking:

Reply:
Right, I had the same thought.
We should either ask UX if it's ok to just omit the label part and only say P31 or we should somehow cut it out before passing it to the api.

Reply:
an executive decision was taken to remove that bit and leave only the id 😄